LDAR_Sim/external_sensors/METEC_WIND.py

```diff
@@ -0,0 +1,154 @@
+#
+# ------------------------------------------------------------------------------
+import numpy as np
+from methods.funcs import measured_rate
+from utils.attribution import update_tag
+from utils.unit_converter import gas_convert
+
+# expects a 3rd value in MDL that is the cutoff point.
+
+
+def detect_emissions(self, site, covered_leaks, covered_equipment_rates, covered_site_rate,
+                     site_rate, venting, equipment_rates):
+    """ 
+    An alternative sensor, specifically built to replicate the probability of detection
+    curves provided by a METEC report, which factors in wind speeds
+
+    Utilizes 3 values set as the MDL:
+        mdl = [a, b, c]
+        where 
+        a and b : are the PoD curve variables 
+        c : represents the floor/minimum cutoff value of the leak rates that the sensor can detect
+
+    PoD = 1 / (1 + e ^ (a - b * r )) 
+
+    a = first MDL value
+    b = second MDL value
+    r = emission rate normalized by wind speed
+
+    Args:
+        site (site obj): Site in which crew is working at
+        covered_leaks (list): list of leak objects that can be detected by the crew
+        covered_equipment_rates (list): list of equipment leak rates that can be
+                                        detected by the crew
+        covered_site_rate (float): total site emissions from leaks that are observable
+                                    from a crew
+        site_rate (float): total site emissions from leaks all leaks at site
+        venting (float): total site emissions from venting
+        equipment_rates (list): list of equipment leak rates for each equipment group
+
+    Returns:
+        site report (dict):
+                site  (site obj): same as input
+                leaks_present (list): same as covered leaks input
+                site_true_rate (float): same as site_rate
+                site_measured_rate (float): total emis from all leaks measured
+                equip_measured_rates (list): total of all leaks measured for each equip group
+                venting (float): same as input
+                found_leak (boolean): Did the crew find at least one leak at the site
+    """
+    missed_leaks_str = '{}_missed_leaks'.format(self.config['label'])
+    equip_measured_rates = []
+    site_measured_rate = 0
+    found_leak = False
+    n_leaks = len(covered_leaks)
+    mdl = self.config['sensor']['MDL']
+
+    hourly_weather = self.state['weather'].get_hourly_weather(
+        ['winds'],
+        self.state['t'].current_date,
+        number_of_hours=1,
+        site=site)
+    average_wind = np.average(hourly_weather['winds']) * 3.6  # m/s to km/hr
+
+    if self.config["measurement_scale"] == "site":
+        # factor of 3.6 converts g/s to kg/h
+        rate_per_wind = covered_site_rate * 3.6 / average_wind
+        prob_detect = 1/(1+np.exp(mdl[0]-mdl[1]*rate_per_wind))
+        if rate_per_wind <= (mdl[2]*3.6):
```

Comment:
Important: Using the wind adjusted rate to determine cutoff instead of the rate without wind will artificially inflate cutoff values. This isn't really an issue as long as users are aware that this is how it works, but we should be crystal clear on this functionality. More documentation is definitely needed and we should consider changing this to not use the wind adjusted rate to remove complexity.

Reply:
Resolved by changing the cutoff to be based on true non-wind adjusted rate. Additional documentation added to the docstring.